run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Is this a script to test the code? Should we be checking it in to the repo?

Let's remove this since it has local file paths that may not be relevant to all users.